otaigbe/Epic-Mail
# #164711536 Protects all endpoints from unauthorized access

#### What does this PR do?
Protects endpoint from unauthorized access
#### Description of Task to be completed?
Install jwt tokens npm dependency and write an authentication middleware
#### How should this be manually tested?
- Clone this Repository
- using git bash terminal cd into the repo
- run npm install and npm start
- Using postman Enter this url into the address bar http://localhost:8833/api/v1/groups
- select the GET method
- Click send
- Response should show unauthorized access.
#### Any background context you want to provide?
Routes need to be protected, and extra information may need to be shared between routes hence the need for the jwt tokens.
#### What are the relevant pivotal tracker stories?
Pivotal tracker id 164711536
